### PR TITLE
Plot reads per prevalence

### PR DESCRIPTION
**Pull request by dp-rice**
Make a new figure (not in the P2RA report) to illustrate the cost estimates.

`./figures/plot_reads_per_prevalence.py` generates `figures/reads_per_prevalence.png`:
reads_per_prevalence

### CODE REVIEW

File: plot_reads_per_prevalence.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            )
+            ax.fill_between(
+                incidence,
+                scale_factor / upper,
+                scale_factor / lower,
+                color=color,
+                alpha=0.2,
+            )
+        ax.set_title(virus)
+        ax.set_xlabel("Prevalence")
```
Comment (jeffkaufman) [inline plot_reads_per_prevalence.py:72]: Maybe supxlabel to share this xlabel between the three plots?

File: plot_reads_per_prevalence.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                max=float(row["max"]),
+            )
+    return data
+
+
+def start():
+    data = read_data()
+    viruses = [
+        ("Norovirus (GII)", "incidence"),
+        ("SARS-COV-2", "incidence"),
```
Comment (jeffkaufman) [inline plot_reads_per_prevalence.py:44]: We'e mostly been doing "Sars-CoV-2"

File: plot_reads_per_prevalence.py (hunk 1 above)
Comment (jeffkaufman) [inline plot_reads_per_prevalence.py:72]: Shouldn't these be "Weekly Incidence" not "Prevalence", at least for the first two?

**Review by jeffkaufman — APPROVED** [review]

File: reads_per_prevalence.png
Comment (jeffkaufman) [inline reads_per_prevalence.png:1]: We don't usually check images into git

File: plot_reads_per_prevalence.py (hunk 1 above)
Comment (dp-rice) [reply plot_reads_per_prevalence.py:72]: They should but this was something Will requested for the original use of the figure (in a report that was eliding the difference)

**Comment by dp-rice** [comment]
@jeffkaufman thanks! For context, this was a quick figure that Will asked for for his report that Simon is planning to adapt for the publication. I've asked him to review the pull request so he's familiar with the code and can ask questions about it.

**Review by simonleandergrimm — APPROVED** [review]
Looks good! I would just move the figure script into the figures directory.

There are some aesthetic details I would change (drop top and right axes, drop ticks on middle and right-most plot), but I can do this when I create a similar figure, as @dp-rice and I discussed yesterday.

**Pull request merged**